app/components/new-sign-up/form.hbs

```diff
@@ -7,14 +7,14 @@
   </div>
 
   <div class="signup-action__btn">
-    <SignUp::Button
-      @clickHandler={{if (eq @state "username")
-      @register @changeRouteParams}}
+    <NewSignUp::Button
```

Comment:
NIT: "New" becomes old in a few weeks.

In future, we should name it to something specific. Maybe "ThreeStepSignUp" or "SignupV2" or something that doesn't use "relative" association.